src/components/open-api-page/index.tsx

```diff
@@ -1,10 +1,9 @@
-import { useCallback, useState, useRef } from 'react'
+import { useRef } from 'react'
```

Comment:
I'm seeing some unused code in here. Do you want to go ahead and remove?

- `setOperationState`
- `isExpanded`

Reply:
I believe I pushed a commit removing these in the middle of your review

Reply:
👍🏻